.github/workflows/zap_templates.yaml

```diff
@@ -44,5 +44,7 @@ jobs:
                 npm run version-stamp
                 npm rebuild canvas --update-binary
                 npm run build-spa
-            - name: Generate
-              run: scripts/tools/zap_generate.sh examples/all-clusters-app/all-clusters-common/all-clusters-app.zap
+            - name: Generate all
+              run: scripts/tools/zap_regen_all.py
+            - name: Check for uncommited changes
+              run: git diff-index --quiet HEAD --
```

Comment:
This will only compare files already tracked by git, right? Should `zap_regen_all.py` have an option to automatically `git add` the generated files to make the workflow detect new ZAPT files products of which were not committed?

Reply:
Good point. I guess I forgot about new files.  I should probably use `git status` instead of `git diff`. Will look at that.

Reply:
I end up using `git add .` as you first suggest.